xclim/indicators/land/_streamflow.py

```diff
@@ -25,6 +31,13 @@ def cfcheck(q):
         check_valid(q, "standard_name", "water_volume_transport_in_river_channel")
 
 
+class StandardizedStreamflowIndexes(ResamplingIndicator):
```

Comment:
Not sure if this should be a new class or `standardized_streamflow_index` should instantiate `StandardizedIndexes` (currently in xclim/indicators/atmos/_precip, but would be relocated in xclim.core.indicator).

Would that break something?

Reply:
I think you're free to create a new class here.